### PR TITLE
Reconstruct segments

### PR DESCRIPTION
**Pull request by exodaniel**
@rdube Review necessary, some small change to how features are stored after being read from file. I've tested this and it seems to be working.

fixes issue #19

### CODE REVIEW

File: segmatch/python/reconstruction_script.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  print("Prescribed:")
+  print("$ python [this_script] [model_path] [segments_path] [features_output_path] [latent_space_dimension]")
+  exit()
+script_path = sys.argv[0]
+model_path = sys.argv[1]
+segments_fifo_path = sys.argv[2]
+features_fifo_path = sys.argv[3]
+latent_space_dim = int(sys.argv[4])
+RC_CONFIDENCE = 0.2
+voxel_side = 24
```
Comment (rdube) [inline segmatch/python/reconstruction_script.py]: side or size?

File: segmatch/python/reconstruction_script.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+try:
+  vae.saver.restore(vae.sess, model_path+"model.checkpoint")
+  print("Model restored.")
+except:
+  raise ValueError("Could not load model.")
+
+print("__INIT_COMPLETE__")
+
+
+os.mkfifo(segments_fifo_path)
```
Comment (rdube) [inline segmatch/python/reconstruction_script.py]: can you add explanatory comment?

File: segmatch_ros/src/segmatch_worker.cpp [hunk 3]
```diff
@@ -305,4 +315,76 @@ bool SegMatchWorker::exportRunServiceCall(std_srvs::Empty::Request& req,
...
+  // Import the autoencoder features from file.
+  LOG(INFO) << "Importing autoencoder segments.";
+  CHECK(database::importSegments(descriptors_params.autoencoder_temp_folder_path + kSegmentsFilename,
+                                 &reconstructed_target_segments));
+  LOG(INFO) << "Done.";
+
+  pclose(script_process_pipe);
+
+  // Move reconstructions to their centroid locations.
+  for (std::unordered_map<Id, Segment>::iterator it = original_target_segments.begin();
```
Comment (rdube) [inline segmatch_ros/src/segmatch_worker.cpp]: `const_iterator`

File: segmatch/python/reconstruction_script.py (hunk 1 above)
Comment (exodaniel) [inline segmatch/python/reconstruction_script.py]: voluntary: a voxel side is 24 units, the voxel size is [24x24x24].

**Pull request merged**